rcl/test/rcl/test_timer.cpp

```diff
@@ -296,7 +296,7 @@ TEST_F(TestTimerFixture, test_canceled_timer) {
     ret = rcl_wait_set_fini(&wait_set);
     EXPECT_EQ(RCL_RET_OK, ret) << rcl_get_error_string().str;
   });
-  ret = rcl_wait(&wait_set, RCL_MS_TO_NS(1));
+  ret = rcl_wait(&wait_set, RCL_MS_TO_NS(100));
```

Comment:
I think this one should be greater than `timer` period (500ms).

Reply:
I misunderstood the test before.

In some sense, these numbers don't matter at all.  We create the timer, and then immediately cancel it, so it will never expire.  From that perspective, we can just do the shortest wait possible (1 millisecond), and it doesn't matter how long that takes; the timer will never expire, so the below stuff will always be fine.  I'm actually going to revert this set of changes as it isn't necessary.